packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -431,7 +461,12 @@ const Dropdown = ({
                 data-testid={test_id}
                 value={value || 0}
             />
-            <div ref={wrapper_ref} className={containerClassName()}>
+            <div
+                ref={wrapper_ref}
+                className={containerClassName()}
+                onMouseOver={handleVisibility}
+                onMouseLeave={handleVisibility}
```

Comment:
will `role='button'` help get rid of SonarCloud issue?

Reply:
In this particular case sonarCloud is waiting for onFocus